src/extension.ts

```diff
@@ -68,6 +68,11 @@ import { randomBytes } from 'crypto';
 const messageBus = new MessageBus();
 
 export async function activate(context: vscode.ExtensionContext) {
+
+	if(!context.storageUri) {
+		throw new Error('Storage Uri not found');
```

Comment:
when can this happen?

Reply:
storageUri has type `uri | undefined`, but im not sure in what cases it can be undefined.